cmd/args.go

```diff
@@ -30,3 +70,31 @@ func ArgValidatorAssetOrP(cmd *cobra.Command, arg string) error {
 	}
 	return fmt.Errorf(errstr)
 }
+
+func ArgValidatorFCTAmount(cmd *cobra.Command, arg string) error {
+	// The FCT amount must not be beyond 1e8 divisible
+	_, err := FactoidToFactoshi(arg)
+	return err
+}
+
+// ArgValidatorECAddress checks for EC address
+func ArgValidatorECAddress(cmd *cobra.Command, arg string) error {
+	if len(arg) > 2 && arg[:2] != "EC" {
+		return fmt.Errorf("EC addresses start with EC")
+	}
+	if err := factoidaddress.ValidEC(arg); err != nil {
+		return fmt.Errorf("%s is not a valid EC address: %s", arg, err.Error())
+	}
+	return nil
+}
+
+// ArgValidatorFCTAddress checks for FCT address
+func ArgValidatorFCTAddress(cmd *cobra.Command, arg string) error {
+	if len(arg) > 2 && arg[:2] != "FA" {
+		return fmt.Errorf("FCT addresses start with FA")
+	}
+	if err := factoidaddress.ValidFCT(arg); err != nil {
+		return fmt.Errorf("%s is not a valid FCT address: %s", arg, err.Error())
+	}
+	return nil
+}
```

Comment:
I'd suggest using AdamSLevy's factom package for these functions. `FAAddress.Set()` and `ECAddress.Set()` IIRC should be able to achieve what you are doing here.

Reply:
Looks like you're using `FactomProject/factom` in your changes. Was this intentional over using the `Factom-Asset-Tokens/factom` package? I thought the former was causing issues for go mod, but maybe I'm misremembering

Reply:
Total accident. Good catch